Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -403,7 +403,7 @@ extension HTTPClient {
             }
         }
 
-        func setChannel(_ channel: Channel) -> Channel {
+        @discardableResult func setChannel(_ channel: Channel) -> Channel {
```

Comment:
could you add newline after `@discardableResult`?

Reply:
@artemredkin Sure! Wondered what was best. Fixed it.